pkg/cmd/roachtest/tests/cdc.go

```diff
@@ -1598,6 +1626,33 @@ func registerCDC(r registry.Registry) {
 			}
 		},
 	})
+	r.Add(registry.TestSpec{
+		Name:             "cdc/kafka-azure",
+		Owner:            `cdc`,
+		CompatibleClouds: registry.AllExceptAWS,
+		Cluster:          r.MakeClusterSpec(2, spec.Arch(vm.ArchAMD64), spec.GCEZones("us-east1-b")),
```

Comment:
Does this test have a dependency to run on amd64 VMs only? If not, then it's best to leave it unspecified; this way it would be eligible for a metamorphic run on arm64.

Reply:
Agreed. Done.